.gitignore

```diff
@@ -2,3 +2,4 @@ passwords.py
 *.pyc
 fafbot.conf
 .coverage
+*.pyc
```

Comment:
Duplicated it now. I could remove the line by clinking the edit button here and thus adding a commit onto the branch. Thinking this probably violates the "clean history" thing going on here. Then again, does having the "Ignore compiled python" commit in here to begin with not do so, since if you rebase correctly, it becomes empty and no longer does what it says it does?